docs/preview/features/logging.md

```diff
@@ -255,6 +257,75 @@ So the resulting log message becomes:
 
 `HTTP Request POST http://localhost:5000/weatherforecast completed with 200 in 00:00:00.0191554 at 03/23/2020 10:12:55 +00:00 - (Context: [X-Api-Key,])`
 
+#### Tracked HTTP status codes
+
+By default, all the responded HTTP status codes will be tracked but this can be altered according to your chosing.
+
+Consider the following API controller. When we configure request tracking, both the `400 BadRequest` response as the `201 Created` response will be tracked.
+
+```csharp
+[ApiController]
+public class OrderController : ControllerBase
+{
+    [HttpPost]
+    public IActionResult Create([FromBody] Order order)
+    {
+        if (order.Id is null)
+        {
+            // Request tracking will happen on this response.
+            return BadRequest("Order ID should be filled out");
+        }
+
+        // Request tracking will happend on this response.
+        return Created($"/orders/{order.Id}", order);
+    }
+}
+```
+
+Let's change the request tracking to only track successful '201 Created' responses.
+This can be changed via the options:
+
+```csharp
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+
+public class Startup
+{
+    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    {
+        app.UseRequestTracking(options => options.TrackedStatusCodes.Add(HttpStatusCode.Created));
```

Comment:
We are using Dapr which add a probe endpoint that we don't control.

In order to filter that out, we would maybe need to define this here somehow based on the path.
How hard would it be to add that?

Reply:
That's more something for another issue and PR, but I'm thinking about routes or route ranges (wildcards) that you could add to the options if you can't control it yourself.
In other scenario's, you could use our `ExcludeRequestTracking` attribute I guess.

Reply:
It's not really the same as we want to limit on a set of status codes, not request/response payload.

Reply:
Seperate task for this is ok though.

Reply:
Ok, not entirely sure what the goal is here, so I suggest that you make the additional issue for this one. 😄 I'll resolve this comment.